### PR TITLE
Added Marathi OpenSLR 64 recipe

### PR DESCRIPTION
**Pull request by SujaySKumar**

### CODE REVIEW

File: egs2/mr_openslr64/asr1/setup.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+../../TEMPLATE/asr1/setup.sh
```
Comment (ftshijt) [inline egs2/mr_openslr64/asr1/setup.sh:1]: This should not be included in specific recipe

File: egs2/mr_openslr64/asr1/local/data.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+. utils/parse_options.sh
+
+log "data preparation started"
+
+mkdir -p ${MARATHI}
```
Comment (ftshijt) [inline egs2/mr_openslr64/asr1/local/data.sh:31]: Could you add a check if the MARATHI is set as empty, it would be dangerous without checking ahead

Comment (SujaySKumar) [reply egs2/mr_openslr64/asr1/local/data.sh:36]: Resolved

File: egs2/mr_openslr64/asr1/setup.sh (hunk 1 above)
Comment (SujaySKumar) [reply egs2/mr_openslr64/asr1/setup.sh]: Resolved

**Comment by SujaySKumar** [comment]
@ftshijt / @sw005320 The recent merging of the `ms_indic_18` recipe to the master branch seems to be breaking the `test_shell` CI check which is failing our above tests

**Comment by sw005320** [comment]
Year, we noticed it and I'm asking Chai, but I don't have a response.
Can you fix it in this PR?

**Comment by sw005320** [comment]
Now we have https://github.com/espnet/espnet/pull/4196

**Comment by SujaySKumar** [comment]
@sw005320 / @ftshijt The `doc/linter_and_test` is failing with a `ImportError: cannot import name 'environmentfilter' from 'jinja2'`. It is not related to any recipes.

How do we fix this CI error? Should we restart the CI?

**Comment by sw005320** [comment]
please ignore this error.

**Comment by SujaySKumar** [comment]
Thanks @sw005320 ! All the other CI tests are passing. This PR can be merged

**Pull request merged**